packages/notebook-extension/src/index.ts

```diff
@@ -1459,7 +1463,18 @@ function addCommands(
       const current = getCurrent(args);
 
       if (current) {
-        return NotebookActions.extendSelectionAbove(current.content);
+        return NotebookActions.extendSelectionAbove(current.content, false);
```

Comment:
`false` is now default, so not needed

Reply:
Ok, I have to change it to the previous one right?

Reply:
Yes, I would just leave the previous code alone, since it should continue to work just fine.